log4j-core/src/main/java/org/apache/logging/log4j/core/package-info.java

```diff
@@ -18,7 +18,7 @@
  * Implementation of Log4j 2.
  */
 @Export
-@Version("2.24.0")
+@Version("3.0.0")
```

Comment:
…revert this change, since the `2.x` branch can not have breaking changes (packages with version `3.0.0`).

Reply:
reverted